ReactiveCocoa/Swift/Signal.swift

```diff
@@ -181,14 +181,24 @@ extension SignalType {
 		return observe(Observer(action))
 	}
 
-	/// Observes the Signal by invoking the given callback when `next` events are
-	/// received.
+	@available(*, unavailable, message="This Signal may emit errors which must be handled explicitly, or observed using observeResult:")
```

Comment:
Marking it as deprecated makes more sense, but remember that Swift doesn't let you call deprecated APIs, so in practice it's the same hehe